lib/src/utils/open_food_api_configuration.dart

```diff
@@ -116,3 +74,50 @@ class OpenFoodAPIConfiguration {
     return null;
   }
 }
+
+/// Uri of the main requests to the backend (OFF).
+const UriProductHelper uriHelperFoodProd = UriProductHelper(
```

Comment:
Wouldn't it be worthwhile to include the other projects (OBF...)?

Reply:
Indeed it would and that's exactly the plan, but it would be a frustrating experience because of #802: `getProduct` does not really work with obf opf opff.